src/tests/ftest/launch.py

```diff
@@ -2123,12 +2249,13 @@ def display_disk_space(path):
         except RunException:
             pass
 
-    def prepare(self, test, repeat):
+    def prepare(self, test, repeat, user_create):
```

Comment:
If this is only called locally then you could rename it to _prepare.

Reply:
I'll defer to @phender on whether this should be `prepare` or `_prepare`, but generally I agree with you.

Reply:
If it's only called from self.prepare then it should be _prepare, if it's called from elsewhere then the name is correct.  Given that you change the arguments and only change the same object that tells me it's not called from elsewhere.

Reply:
I think we aren't in the mindset of always naming "private" functions appropriately, but we should be.

Reply:
This is clearly defined in the python docs, it's appropriate to change things to private if you've identified something that can be.

https://docs.python.org/3.8/tutorial/classes.html#private-variables

Reply:
Agree. Done